web/pdf_find_controller.js

```diff
@@ -272,12 +275,11 @@ class PDFFindController {
   _calculateMatch(pageIndex) {
     let pageContent = this._normalize(this._pageContents[pageIndex]);
     let query = this._normalize(this.state.query);
-    let caseSensitive = this.state.caseSensitive;
-    let phraseSearch = this.state.phraseSearch;
+    const caseSensitive = this.state.caseSensitive;
+    const phraseSearch = this.state.phraseSearch;
     const entireWord = this.state.entireWord;
```

Comment:
Nit: This above three lines could be `const { caseSensitive, entireWord, phraseSearch, } = this.state;` instead, if you wanted to :-)

Reply:
Done in the new commit series.